doc/compodoc_sources/how-to-guides/reports.md

```diff
@@ -0,0 +1,90 @@
+# Reports
+The reporting module allows organizations to automatically create reports of aggregated data for a given timespan.
+This can be used to track indicators and export anonymous data.
```

Comment:
@Schottkyc137 Does this give a good introduction?

Reply:
This is what I had in mind, nice work!